geoutils/projtools.py

```diff
@@ -19,6 +19,76 @@
 from geoutils.geovector import Vector
 
 
+def latlon_to_utm(lat: float, lon: float) -> str:
```

Comment:
This whole function could be done with one long pyproj line:
```python
>>> lat = 78
>>> lon = 15
>>> pyproj.database.query_utm_crs_info("WGS 84", area_of_interest=pyproj.aoi.AreaOfInterest(lon, lat, lon, lat))[0].name.split(" ")[-1]
"33N"
```
Might save a few headaches down the line since most logic is handled by pyproj instead. No need to reinvent the wheel!

Reply:
Updated with this, thanks!!